monitor/dbt_project/models/alerts/alerts.sql

```diff
@@ -0,0 +1,15 @@
+{{
+  config(
+    materialized = 'incremental',
+    unique_key = 'alert_id',
+    merge_update_columns = ['alert_id'],
+    on_schema_change = 'append_new_columns'
+  )
+}}
+
+
+select *, false as alert_sent from {{ ref('elementary', 'failed_tests') }}
```

Comment:
This won't work with old package versions. I'd avoid adding this middle view for now and implementing its content here